module/x/gravity/types/key.go

```diff
@@ -41,15 +40,6 @@ var (
 	// [0x2f522701d7f28af84632f8228fbe1750]
 	ValsetConfirmKey = HashString("ValsetConfirmKey")
 
```

Comment:
Several of these keys were not being used after various migrations, so I removed them. Their legacy is in the x/gravity/migrations folder

Reply:
Undid the removal, renamed the unused keys to LEGACY*